inc/TranslationApiRoute.php

```diff
@@ -50,7 +50,7 @@ public function route_callback( $project_path ) : void {
 
 			$zip_provider = new ZipProvider( $set );
 
-			if ( ! file_exists( $zip_provider->get_zip_path() ) ) {
+			if ( ! file_exists( $zip_provider->get_zip_path() ) || ! $zip_provider->get_last_build_time() ) {
```

Comment:
Since `file_exists()` is a disk I/O operation I'd switch the conditions to check first for the last build time.